code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -23,6 +23,8 @@
 		return
 	A.attack_ghost(src)
 	if(ishuman(A) && in_range(src, A))
+		if(isLivingSSD(A) && client?.send_ssd_warning(A)) //Do NOT Harvest SSD people unless you accept the warning
```

Comment:
```suggestion
		if(isLivingSSD(A) && client.send_ssd_warning(A)) //Do NOT Harvest SSD people unless you accept the warning
```
Minor thing. `client` can't be null here.

Reply:
If it can't be null then yeah, in it goes!